packages/web-components/wrapper-bindings/angular-proxies-list-ignore.ts

```diff
@@ -47,7 +47,7 @@ export const angularProxiesListIgnore = (
             compilerCtx.events.on('buildLog', (log) => {
                 if (
                     log.messages.findIndex((elm) =>
-                        elm.includes('generate react-boolean-fix finished')
+                        elm.includes('generate react-library finished')
```

Comment:
what happened here? find/replace?

Reply:
I had to change this when I was testing to see if the react boolean issue was fixed, since it previously relied on the boolean fix to run first. Will change it back